test/js/css-image-value.js

```diff
@@ -0,0 +1,34 @@
+suite('CSSImageValue', function() {
+  test('CSSImageValue object is a CSSImageValue, CSSResourceValue, and CSSStyleValue', function() {
+    assert.instanceOf(new typedOM.internal.CSSImageValue(new Image()), typedOM.internal.CSSImageValue);
```

Comment:
I'm still not sure. So should I still make the constructor that accepts an Image internal?